Package.swift

```diff
@@ -17,12 +17,16 @@ let package = Package(
             targets: ["Networking"]
         )
     ],
-    dependencies: [.package(url: "https://github.com/realm/SwiftLint.git", exact: "0.53.0")],
+    dependencies: [
+        .package(url: "https://github.com/realm/SwiftLint.git", exact: "0.53.0"),
+        .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0")
+    ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
         // Targets can depend on other targets in this package, and on products in packages this package depends on.
         .target(
             name: "Networking",
+            swiftSettings: [.unsafeFlags(["-emit-extension-block-symbols"])],
```

Comment:
Could you forward somewhere where I can read more about it? 😄

Reply:
Yeah so the problem was, that I wasn't able to reference extensions of types which are not defined in this module. So for example when I wanted to reference ``URLSessionTask.DownloadState`` it would not create the link. I found the solution here https://github.com/apple/swift-docc/issues/210#issuecomment-1343975239